plugins/modules/junos_l2_interface.py

```diff
@@ -90,6 +88,11 @@
 extends_documentation_fragment:
 - junipernetworks.junos.junos
 
+
+
+
+
+
```

Comment:
Is this space required?

Reply:
@ganeshrn definitely not required.  thanks for finding this, I'll correct this.